README.md

```diff
@@ -1 +1,11 @@
-# learn-css
\ No newline at end of file
+# learn-css
+
+## learn-sass
+
+Learning how to use scss styles in html pages.
+
+### Generate `css` File
+
+Since html pages accept `.css` format as a style, we used the `sass` module to
+convert the scss file to css. We gave the created `.css` file to the html page
+with a link.

```

Comment:
we prefer `scss` syntax over `sass` syntax gibi bir sey diyelim
we prefer `npm` installation instead of `brew` or `choco` to have a simple single run file
we use `npx sass` for convenience

gibi bir seyler karalayin